src/states/OutstreamDone.js

```diff
@@ -0,0 +1,39 @@
+import videojs from 'video.js';
+import States from '../states.js';
+
+const AdState = States.getState('AdState');
+
+/**
+ * This is the final state for a player in outstream mode. There
+ * should be no more ads playing once the player has transitioned
+ * to this state.
+ */
+class OutstreamDone extends AdState {
+
+  /**
+   * Allows state name to be logged after minification
+   */
+  static _getName() {
+    return 'OutstreamDone';
+  }
+
+  /*
+   * For state transitions to work correctly, initialization should
+   * happen here, not in a constructor.
+   */
+  init(player) {
+    player.ads._contentHasEnded = true;
```

Comment:
I understand that we may want this for consistency but is `_contentHasEnded` relevant for outstream players where there's no video content to scrub back through?

Reply:
That's true actually, I was doing it for consistency but it doesn't really make sense to have this - removing it now 😁